src/app/users/views/editServices.ejs

```diff
@@ -32,7 +32,7 @@
 
                         <% if (locals.serviceRoles.length > 0) { %>
                             <legend class="govuk-fieldset__legend govuk-fieldset__legend--xs">
-                                <p class="govuk-body">Some services ask you to select a sub-service, role, or form to better define how you will use them.
+                                <p class="govuk-body"><%= locals.serviceRoles.length > 1 ? 'Select a minimum of 1 role.' : 'Select 1 role.' %> Any changes you make to sub-service selection will change access for <%= locals.user.firstName %> <%= locals.user.lastName %>.
```

Comment:
If a service has multiple roles available but allows only one role to be selected, the user will initially be prompted to select a minimum of one role. However, if the user selects more than one role, it would be considered invalid according to the service's requirements. Is this the right behaviour we're expecting in here?